ERROR - 2025-09-19 04:32:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:32:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:32:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:32:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:33:26 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:33:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:33:29 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:33:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:33:30 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:33:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:33:30 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:33:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:33:30 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:33:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:34:43 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:34:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:36:50 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2025-09-19 04:36:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: testdb]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\TestDb.php(12): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TestDb->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\my-ci-app\\backend\\public\\index.php')
ERROR - 2025-09-19 04:40:25 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'budget_hmmi', 3306, '', 0)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(22): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#6 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#8 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\laragon\www\my-ci-app\backend\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\my-ci-app\backend\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\my-ci-app\backend\app\Controllers\TestDb.php(22): CodeIgniter\Database\BaseConnection->query('SHOW TABLES')
#3 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(933): App\Controllers\TestDb->index()
#4 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TestDb))
#5 C:\laragon\www\my-ci-app\backend\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\my-ci-app\backend\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#7 C:\laragon\www\my-ci-app\backend\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\laragon\www\my-ci-app\backend\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\laragon\www\my-ci-app\backend\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#10 {main}
